ops/terraform/services/static-site/main.tf

```diff
@@ -75,6 +75,5 @@ locals {
   static_site_fqdn       = "${local.env}.static.${local.root_domain_name}"
```

Comment:
given that the name for the acquired certificate is "test.bfd.cmscloud.local", line 75 will need the corresponding update (i.e. remove "static.")

Reply:
Good catch. I updated this in [0c8f773](https://github.com/CMSgov/beneficiary-fhir-data/pull/2411/commits/0c8f773e6e03417cfffafc264bea93eaaff95ae3).